### ISSUE TITLE
How to set the location for CachedImages? 

### ISSUE THREAD

**Issue by owlstack** [luberda-molinet/FFImageLoading#230]
I want to set a specific location to store the CachedImages but I'm not sure how to set the location for the images. It looks like currently, Android is sweeping the bitmaps as soon as I set them for preloading and only shows the first two images that are shown on the page but not the others.... they are collecting and freed for memory.

**Comment by molinch** [comment]
@owlstack I am sorry but I have not understood your need. Could you rephrase?

**Comment by owlstack** [comment]
It's okay, where are the cached Images currently stored on the android device? I was wondering if I can change the location of storage and how do I set the change of location to store the images.

**Comment by molinch** [comment]
By default it creates a subfolder `FFSimpleDiskCache` and stores images there.
But you can provide your own disk cache too using `ImageService.Initialize`.

If you provide your disk cache then you can create an instance of `SimpleDiskCache`, when doing so you can specify your path.
Please note that file names are hashed based on request.

**Comment by owlstack** [comment]
Okay that sounds right. Do you have an example I can reference? Thanks!

**Comment by molinch** [comment]
Something like (not tested):
```c#
			var config = new FFImageLoading.Config.Configuration()
			{
				DiskCache = new SimpleDiskCache(yourPathHere)
			};
			ImageService.Instance.Initialize(config);
```
